apps/condo/domains/organization/hooks/useCreateOrganizationModalForm.tsx

```diff
@@ -58,11 +59,15 @@ const prepareFinishFetchParams = ({ id, userId }) => ({
         },
     },
 })
-const prepareValidationErrorsMapping = ({ ValueIsTooShortMsg, TinTooShortMsg, TinValueIsInvalid }) => ({
-    [EMPTY_NAME_ERROR]: {
+const prepareValidationErrorsMapping = ({ ValueIsTooShortMsg, OrganizationNameIsInvalid, TinTooShortMsg, TinValueIsInvalid }) => ({
```

Comment:
Look at error examples in that list. You trying to mix old style errors with new one

Reply:
fixed